### PR TITLE
Remove population-wide HSI event in ChronicSyndrome module

### PR DESCRIPTION
**Pull request by marghe-molaro**
As raised in Issue #1260, our assumptions around how the healthsystem handles resource constraints does not allow for the use of Population-wide HSI events. Here we remove one such instance from the mock ChronicSyndrome module used by some of the HealthSystem tests.

### CODE REVIEW

File: src/tlo/methods/chronicsyndrome.py [hunk 1]
```diff
@@ -214,11 +214,11 @@ def initialise_simulation(self, sim):
         self.sim.schedule_event(outreach_event, self.sim.date + DateOffset(months=6))
 
         # Schedule the occurance of a population wide change in risk that goes through the health system:
```
Comment (matt-graham) [inline src/tlo/methods/chronicsyndrome.py:216]:
```suggestion
        # Schedule the occurrence of a population wide change in risk:
```

Possibly remove the reference to health system in comment as no longer a HSI event?

**Review by matt-graham — COMMENTED** [review]
With regards to #1260 - would the plan be to remove the logic in the health system scheduler for processing population scoped HSI events in a separate PR? It might also be good to add some logic to `HSI_Event` or `PopulationScopeEventMixin` in such a PR that raises an error when trying to subclass from both if so to ensure we fail loudly on trying to instantiate a population-scoped HSI event.

**Comment by tbhallett** [comment]
> With regards to #1260 - would the plan be to remove the logic in the health system scheduler for processing population scoped HSI events in a separate PR? It might also be good to add some logic to `HSI_Event` or `PopulationScopeEventMixin` in such a PR that raises an error when trying to subclass from both if so to ensure we fail loudly on trying to instantiate a population-scoped HSI event.

Yes I think that's a good idea. @marghe-molaro do you want to do that, or shall I?

**Comment by marghe-molaro** [comment]
Hi @tbhallett and @matt-graham, 
The reason I didn't include that 'loud error' is that I'm a bit concerned that this might lead to failure during runtime if there are rare population-wide HSI events which are missed by tests. I think the safer option would be to remove the option for a pop-wide HSI completely, to make sure they cannot be used anywhere.

What do you think?

**Comment by tbhallett** [comment]
> Hi @tbhallett and @matt-graham, The reason I didn't include that 'loud error' is that I'm a bit concerned that this might lead to failure during runtime if there are rare population-wide HSI events which are missed by tests. I think the safer option would be to remove the option for a pop-wide HSI completely, to make sure they cannot be used anywhere.
> 
> What do you think?

I reckon we should take the plunge now (and in this PR) to remove the possibility of "Population-wide" HSI in the HealthSystem code entirely. (It clutters up the code, and going-forwards, it complicates things, and we don't want anyone to implement a new one!). We'll do tests to see if there is any other other population-HSI lurking. As @marghe-molaro proposes changes to accommodate the results of her emulation models etc, it seems a good time to make this change and set a cleaner baseline.

**Comment by tbhallett** [comment]
> Just adding a note here that actually the original motivation for removing Pop-wide HSI events is that it would be difficult to enforce HCW time resource constraints on them. Working on it now!

Yes, that's quite right.

**Comment by tbhallett** [comment]
Closing this as this is now done, together with wider changes to remove Population level HSI in #1557

**Pull request closed without merging**